src/components/DatePicker/DatePicker.utils.ts

```diff
@@ -0,0 +1,8 @@
+import { format } from 'date-fns';
+
+export const getDatesStringsArray = (value: string) =>
+  value?.split('-').map((element) => element.trim());
+
+export const getStringFromDate = (date: Date) => String(format(date, 'dd MMM yyyy'));
```

Comment:
maybe it's better to create constant for date format

Reply:
done